docs/reference/commandline/stack_deploy.md

```diff
@@ -92,8 +92,8 @@ axqh55ipl40h  vossibility_vossibility-collector  replicated  1/1       icecrime/
 
 ## Related information
 
+* [deploy](deploy.md)
```

Comment:
I think we can just remove `deploy` here, because it's an alias for `stack deploy` (which is the page we're currently on)